sql/core/src/main/scala/org/apache/spark/sql/execution/arrow/ArrowConverters.scala

```diff
@@ -92,6 +92,9 @@ private[sql] object ArrowConverters extends Logging {
     private val root = VectorSchemaRoot.create(arrowSchema, allocator)
     protected val unloader = new VectorUnloader(root)
     protected val arrowWriter = ArrowWriter.create(root)
+    private var _lastBatchRowCount: Long = -1L
```

Comment:
Let's don't use `_` prefix. Just use the name like `lastBatchRowCount`, and `getLastBatchRowCount`

Reply:
Let's name it `rowCountInLastBatch` to be consistent with `ArrowBatchWithSchemaIterator` below.